installed Tailwindcss and StimulusJS

desoto13 opened the pull request:
1. Installed StimulusJS to our forked stock_app
2. Installed TailwindCSS as our CSS framework

desoto13 committed:
installed Tailwindcss and StimulusJS

analyn-cajocson commented on app/javascript/controllers/index.js:
```diff
@@ -0,0 +1,6 @@
+
+const application = Application.start(document.documentElement)
+const context = require.context('.', true, /_controller\.js$/)
+application.load(definitionsFromContext(context))
```
Add new line at the end of file

analyn-cajocson commented on app/javascript/stylesheets/application.scss:
```diff
@@ -0,0 +1,5 @@
+@import "tailwindcss/components";
+//Custom styles here
+
+@import "tailwindcss/utilities";
```
add new line at the end of file

desoto13 committed:
added a new line at the end of the file

desoto13 commented:
1. Added a new line at the end of the file app/javascript/controllers/index.js
2. Added a new line at the end of the file app/javascript/stylesheets/application.scss
